cmd/install/testdata/zz_fixture_TestHelmCommand_values.yaml

```diff
@@ -0,0 +1,17 @@
+aws:
```

Comment:
We'll want to have test cases to validate these fields are actually populated as you would expect. Otherwise the value of the test is quite limited. I'm ok if we want to do that in a follow up as we move forward.